Fix/cs/updates 1 (#112)
uc-cdis/gen3.org: content/resources/user/query-data.md

}
```

> Queries like `(first: 0)` are not supported. To return more entries, the query call can specify a larger number such as `(first: 100)`. The maximum number of results returned is 10,000 (`max_result_window`). If you need to access more than that number, we suggest using the /guppy [download endpoint](https://github.com/uc-cdis/guppy/blob/master/doc/download.md).

Reviewer: ```suggestion
To return more entries, add an argument to the query call, such as `(first: 100)`. The maximum number of results returned is 10,000 (`first: 10000`). If you need to access more than that number, we suggest using the /guppy [download endpoint](https://github.com/uc-cdis/guppy/blob/master/doc/download.md).
```
